### DISCUSSION TITLE
Deprecation of `POST /notifications` in favor of `POST /broadcasts`?

### DISCUSSION THREAD

**Discussion by cdoane-evo** [magicbell/community#166] [Questions] [answered]
I just wanted to confirm as I couldn't find a note or changelog about it on the docs, but in the API has `POST /notifications` been replaced by `POST /broadcasts`?

If so is there any timeline to turn off support for `POST /notifications` so we can schedule the work to migrate?

**Comment by unamashana** [comment, marked as answer]
Hey @cdoane-evo thanks for spotting this! We don't plan to deprecate this endpoint anytime soon, and if we do, we will offer sufficient notice. In the backend, we've been able to make both endpoints work off the same path (with some exceptions, and therefore, omissions in the new endpoint).
